rmgpy/chemkinTest.py

```diff
@@ -118,25 +118,26 @@ def test_readThermoEntry_NoTRange(self):
         self.assertEqual(formula, {'H': 6, 'C': 2})
         self.assertTrue(isinstance(thermo, NASA))
 
-    def testReadAndWriteTemplateReactionFamilyForMinimalExample(self):
+    def testReadAndWriteAndReadTemplateReactionFamilyForMinimalExample(self):
```

Comment:
I'm not clear on why this is a better name for this function

Reply:
I added an additional read step, so I thought I'd change the name to mention that. Should I change it back?

Reply:
I guess this is fine.  I find it kind of hard to read with all the and's, but it is a precise name.